docs/spmd.md

```diff
@@ -270,6 +270,56 @@ dist_cp.load_state_dict(
 model.load_state_dict(state_dict["model"])
 ```
 
+#### CheckpointManager
+
+The experimental [CheckpointManager](https://github.com/pytorch/xla/blob/master/torch_xla/experimental/distributed_checkpoint/manager.py#L40)
```

Comment:
does this feature tie to the SPMD? If not I felt like it should be in a separate doc.

Reply:
We currently only support `torch.distributed.checkpoint` with SPMD, but CheckpointManager only builds on `torch.distributed.checkpoint` and isn't SPMD-specific.

Reply:
Do we have any plan to support non-spmd use cases? How difficult will it be?

Reply:
I guess that's nice to have to support non-SPMD case, but I guess the future is to move everything to be on top of SPMD?

Reply:
Sorry missed this yesterday - I don't know of any plans to support non-SPMD use cases like user-mode FSDP, but AFAIK it will require an additional planner implementation.

Since the usage is restricted to SPMD at the moment, I'd say we should keep it in the SPMD doc for now and consider moving it out when the API becomes more stable or we add more use cases, wdyt @JackCaoG?